pdns/pdns_recursor.cc

```diff
@@ -4205,6 +4310,20 @@ static int serviceMain(int argc, char*argv[])
   g_proxyProtocolACL.toMasks(::arg()["proxy-protocol-from"]);
   g_proxyProtocolMaximumSize = ::arg().asNum("proxy-protocol-maximum-size");
 
+  if (!::arg()["dns64-prefix"].empty()) {
+    auto dns64Prefix = Netmask(::arg()["dns64-prefix"]);
```

Comment:
the Netmask ctor can throw a NetmaskException in `makeComboAddress`, please try/catch this to prevent abort()